File: backend-node-api/models/artwork.js
/**
 * Represents a painting.
 * @class
 */
export class Artwork {
  /**
   * Creates a new artwork.
   *
   * @param {string} title - The title of the painting.
   * @param {string} artist - The artist who created the painting.
   * @param {number} year - The year the painting was created.
   * @param {number} weight - The weight of the painting in *grams.
   */
  constructor(title, artist, year, weight) {
    this.title = title;
    this.artist = artist;
    this.year = year;
    this.weight = weight;
  }

File: backend-java-api/src/main/java/com/example/backend_java_api/models/Artwork.java
package com.example.backend_java_api.models;

public class Artwork {

    private String title;
    private String artist;
    private int year;
    private int weight; // weight in grams

    // Constructor for Artwork
    public Artwork(String title, String artist, int year, int weight) {
        this.title = title;
        this.artist = artist;
        this.year = year;
        this.weight = weight;
    }

    // Getters for the properties
    public String getTitle() {
        return title;
    }

    public String getArtist() {
        return artist;
    }

    public int getYear() {
        return year;
    }

    public int getWeight() {
        return weight;
    }
}
